ghmirror/data_structures/redis_data_structures.py

```diff
@@ -25,6 +25,8 @@
 REDIS_PORT = os.environ.get('REDIS_PORT', 6379)
 REDIS_TOKEN = os.environ.get('REDIS_TOKEN')
 REPLICA_ENDPOINT = os.environ.get('REPLICA_ENDPOINT')
+if REPLICA_ENDPOINT == '':
+    REPLICA_ENDPOINT = REDIS_ENDPOINT
 
```

Comment:
@apahim what do you think of this? This will allow github mirror to run with a single Redis instance or a cluster with a primary and read only replicas. Otherwise the user will be forced to run at least two instances.

Reply:
I read your previous comment more closely now. so nevermind this!